Program.cs

```diff
@@ -15,6 +15,7 @@ public Item(string itm, string loc)
                 location = loc;
             }
         }
+        public static int playerHP = 20;
```

Comment:
it would be cool to know what level of HP the player and foe after each attack and before as well.

Reply:
Decided not to add this as I think an in game HP gauge removes from the realism of the fight

Reply:
That is fair. I will mark this as resolved now.